lib/measures/tbd/resources/psi.rb

```diff
@@ -61,79 +64,256 @@ def append(k)
+      party:         0.850, # *
+      grade:         0.850, # *
+      joint:         0.300, # *
+      transition:    0.000
+    }.freeze               # based on INTERIOR dimensions (p.15 BETBG)
+    self.genShorthands("poor (BETBG)")
+
+    @set["regular (BETBG)"] =
     {
-      rimjoist:     0.500, # *
-      parapet:      0.450, # *
-      fenestration: 0.350, # *
-      concave:      0.450, # *
-      convex:       0.450, # *
-      balcony:      0.500, # *
-      party:        0.500, # **
-      grade:        0.450  # *
-    }.freeze
-
-    @set[ "efficient (BC Hydro)" ] =
+      rimjoist:      0.500, # *
+      parapet:       0.450, # *
+      fenestration:  0.350, # *
+      corner:        0.450, # *
+      balcony:       0.500, # *
+      party:         0.450, # *
+      grade:         0.450, # *
+      joint:         0.200, # *
+      transition:    0.000
+    }.freeze               # based on INTERIOR dimensions (p.15 BETBG)
+    self.genShorthands("regular (BETBG)")
+
+    @set["efficient (BETBG)"] =
     {
-      rimjoist:     0.200, # *
-      parapet:      0.200, # *
-      fenestration: 0.200, # *
-      concave:      0.200, # *
-      convex:       0.200, # *
-      balcony:      0.200, # *
-      party:        0.200, # *
-      grade:        0.200  # *
-    }.freeze
-
-    @set[ "code (Quebec)" ] = # NECB-QC (code-compliant) defaults:
+      rimjoist:      0.200, # *
+      parapet:       0.200, # *
+      fenestration:  0.200, # *
+      corner:        0.200, # *
+      balcony:       0.200, # *
+      party:         0.200, # *
+      grade:         0.200, # *
+      joint:         0.100, # *
+      transition:    0.000
+    }.freeze               # based on INTERIOR dimensions (p.15 BETBG)
+    self.genShorthands("efficient (BETBG)")
+
+    @set["spandrel (BETBG)"] =
     {
-      rimjoist:     0.300, # *
-      parapet:      0.325, # *
-      fenestration: 0.350, # **
-      concave:      0.450, # **
-      convex:       0.450, # **
-      balcony:      0.500, # *
-      party:        0.500, # **
-      grade:        0.450  # **
-    }.freeze
-
-    @set[ "(non thermal bridging)" ] = # ... would not derate surfaces:
+      rimjoist:      0.615, # * Detail 1.2.1
+      parapet:       1.000, # * Detail 1.3.2
+      fenestration:  0.000, # * ... generally part of clear-field RSi
+      corner:        0.425, # * Detail 1.4.1
+      balcony:       1.110, # * Detail 8.1.9/9.1.6
+      party:         0.990, # ** ... similar to parapet/balcony
+      grade:         0.880, # * Detail 2.5.1
+      joint:         0.500, # * Detail 3.3.2
+      transition:    0.000
+    }.freeze               # "conventional", closer to window wall spandrels
+    self.genShorthands("spandrel (BETBG)")
+
+    @set["spandrel HP (BETBG)"] =
+    {
+      rimjoist:      0.170, # * Detail 1.2.7
+      parapet:       0.660, # * Detail 1.3.2
+      fenestration:  0.000, # * ... generally part of clear-field RSi
+      corner:        0.200, # * Detail 1.4.2
```

Comment:
No concavecorner?

Reply:
In this case, TBD will tag corner edges in an OSM as either _concave_ or _convex_ - both of which will inherit the simple _corner_ PSI value of 0.200 W/K per m. If one has access to PSI data for a given construction system, differentiating _concave_ vs _convex_ corners (e.g. 0.150 vs 0.250), then one can add _cornerconcave_ and _cornerconvex_ items in the PSI set.